### PR TITLE
Improve MiniYaml nunit tests

### LINKED ISSUE #15155: OpenRA.Test/OpenRA.Game/MiniYamlTest.cs(8,8): Warning CS0219: The variable `resultKeep' is assigned but its value is never used (CS0219) (OpenRA.Test)
https://github.com/OpenRA/OpenRA/blob/6fdb25b48f86f8fe55edd15c3579ec16ed669f74/OpenRA.Test/OpenRA.Game/MiniYamlTest.cs#L237-L238

### PR DESCRIPTION
**Pull request by pchote**
Fixes #15155.

### CODE REVIEW

**Comment by pchote** [comment]
Fixed code style warning.

File: OpenRA.Test/OpenRA.Game/MiniYamlTest.cs [hunk 1]
```diff
@@ -230,12 +230,15 @@ public void CommentsShouldCountTowardLineNumbers()
...
-			var resultKeep = MiniYaml.FromString(yaml, discardCommentsAndWhitespace: false).First(n => n.Key == "TestB");
-			Assert.That(resultDiscard.Location.Line, Is.EqualTo(6), "Node TestB should report its location as line 6, but is not (parsing comments)");
+			var resultDiscard = MiniYaml.FromString(yaml);
+			var resultDiscardLine = resultDiscard.First(n => n.Key == "TestB").Location.Line;
+			Assert.That(resultDiscardLine, Is.EqualTo(6), "Node TestB should report its location as line 6, but is not (discarding comments)");
+			Assert.That(resultDiscard[1].Key, Is.EqualTo("TestB"), "Node TestB should be the second child of the root node, but is not (discarding comments)");
+
+			var resultKeep = MiniYaml.FromString(yaml, discardCommentsAndWhitespace: false);
+			var resultKeepLine = resultKeep.First(n => n.Key == "TestB").Location.Line;
+			Assert.That(resultKeepLine, Is.EqualTo(6), "Node TestB should report its location as line 6, but is not (discarding comments)");
```
Comment (abcdefg30) [inline OpenRA.Test/OpenRA.Game/MiniYamlTest.cs]: Shouldn't that be a `(parsing comments)` as well?

**Comment by pchote** [comment]
Fixed.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/2b1d73eb511013e0bf820a1d0ffd770ce18b9324...0af46c31fe600207c443c1f20bb7a1e9ebe85ddf)

**Pull request merged**